### PR TITLE
feat: GEOT-351: styles for promoted tags below search bar

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/app-andrew/src/components/group-settings/PromotedTagForm/index.tsx [hunk 1]
```diff
@@ -115,101 +116,98 @@ export const PromotedTagsForm: FC<{
...
-            <button
-              onClick={() => setStep(FormSteps.one)}
-              className={styles.btn}
-            >
-              Go back to step one
-            </button>
-          </div>
-        </section>
+          <button onClick={() => setStep(FormSteps.one)} className={styles.btn}>
+            {/* @todo add translation */}
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group-settings/PromotedTagForm/index.tsx:207]: do not add this comments again, its obvious that we need to add translations here

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss [hunk 2]
```diff
@@ -1,50 +1,85 @@
...
 
-    picture {
-      float: left;
-      margin: 0 10px 5px 0;
-    }
+      .actionButtons {
+        position: absolute;
+        top: 10px;
+        right: 0;
+        z-index: 11;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:46]: why this z-index has this value?

File: packages/app-andrew/src/pages/group/[slug]/settings.tsx [hunk 3]
```diff
@@ -76,136 +76,135 @@ export const GroupSettingsPage: NextPage<GroupSettingsPageProps> = ({
...
-            <Spinner />
-          </div>
-        )}
+        {group.isLoading && !group.errorOccured && <Spinner />}
 
         {!group.isLoading && group.errorOccured && (
-          <div>
-            <p>Can not fetch group details, try again later</p>
-          </div>
+          <p>Can not fetch group details, try again later</p>
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[slug]/settings.tsx:91]: we need to have proper UI for errors as errors may and will happen for many reasons and UI has to be ready for that

File: packages/app-andrew/src/components/group-settings/PromotedTagForm/index.tsx (hunk 1 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group-settings/PromotedTagForm/index.tsx:237]: ok

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss (hunk 2 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:46]: fixed

File: packages/app-andrew/src/pages/group/[slug]/settings.tsx (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[slug]/settings.tsx:91]: will be done in next PRs

**Pull request merged**